.travis.yml

```diff
@@ -29,6 +29,7 @@ notifications:
     on_failure: change
 
 before_install:
+- bash scripts/install-pip.sh
```

Comment:
AFAICT, Travis was working fine before without this. Why is the change needed now?